docs/_static/css/custom_styles.css

```diff
@@ -912,4 +912,109 @@ a.custom-link:hover {
 
 
 
+/* Change button at the top of installation guide */
 
+
+/* Orange version */
+/* .sd-sphinx-override.sd-btn.sd-text-wrap.sd-btn-info
+{
+  
+  background-color: var(--dtu-orange) !important;
+  border: 3px solid var(--dtu-orange)  !important;
+  
+  border-radius: 12px !important;
+  margin-bottom: -20px !important;
+  color:#DADADA !important;
+  
+  
+
+} */
+
+
+/* Grey version - white text */
+/* .sd-sphinx-override.sd-btn.sd-text-wrap.sd-btn-info
+{
+  
+  background-color: #c5c1c1 !important;
+  border: 3px solid #c5c1c1  !important;
+  
+  border-radius: 12px !important;
+  margin-bottom: -20px !important;
+  color:white !important;
+  
+  
+
+} */
+
+/* Grey version - black text */
+
+.sd-sphinx-override.sd-btn.sd-text-wrap.sd-btn-primary.reference.external
+{
+  background-color: #f3f4f5 !important;
```

Comment:
could any of these colors be references to the DTU colors? i.e. something like DTU gray `/ 50%` or something similar?